pkg/velaql/providers/query/collector.go

```diff
@@ -56,6 +57,8 @@ func NewAppCollector(cli client.Client, opt Option) *AppCollector {
 	}
 }
 
+const velaVersionNumberToUpgradeVelaQL = "v1.2.0-rc.1"
```

Comment:
why there's a hardcode version number here?

Reply:
For compatibility, if the vela version is less than this version, we will use the old query method